morgoth/addons/models.py

```diff
@@ -74,6 +77,12 @@ def __str__(self):
     def __repr__(self):
         return '<VersionNumber: {}>'.format(self)
 
+    def __eq__(self, other):
+        return isinstance(other, VersionNumber) and int(self) == int(other)
```

Comment:
This returns `False` if the version is being compared to a non-Version object, when it really should be raising a `TypeError`.